src/app/components/create-experiment/create-experiment.component.ts

```diff
@@ -64,7 +66,7 @@ export class CreateExperimentComponent implements OnInit {
         queryParams: {
           deviceId: this.experimentForm.controls.deviceId.value,
           description: this.experimentForm.controls.description.value,
-          startTime: this.experimentForm.controls.startTime.value,
+          startTime: this.experimentForm.controls.startTime.value ? this.experimentForm.controls.startTime.value.valueOf() : null,
```

Comment:
I'm surprised that .value needs to be followed by a .valueOf() XD

Reply:
The value stored in the form is a moment. We want the query parameter to be in milliseconds. `valueOf()` returns the milliseconds from a moment object